graders/python/python_autograder/code_feedback.py

```diff
@@ -304,6 +304,110 @@ def bad(msg):
 
         return True
 
+    @classmethod
+    def check_dict(
+        cls,
+        name,
+        ref,
+        data,
+        partial_keys=None,
+        check_keys=False,
+        check_values=False,
+        key_type=None,
+        value_type=None,
+        accuracy_critical=False,
```

Comment:
I don't think the name `accuracy_critical` describes the flag accurately; perhaps `abort_invalid=True|False`? I'm not a fan of that new name much better, maybe @eliotwrobson you have another suggestion?

Reply:
It looks like this is a convention from other functions; so I will resolve this.